src/map/clif.cpp

```diff
@@ -11304,6 +11304,7 @@ void clif_parse_WalkToXY(int fd, map_session_data *sd)
 		skill_check_cloaking(&sd->bl, sd->sc.getSCE(SC_CLOAKING));
 	status_change_end(&sd->bl, SC_ROLLINGCUTTER); // If you move, you lose your counters. [malufett]
 	status_change_end(&sd->bl, SC_CRESCIVEBOLT);
+	status_change_end(&sd->bl, SC_KI_SUL_RAMPAGE);
```

Comment:
Would it not be better to move all those into unit_walktoxy (like in the original PR)?
Then it would work correctly also for non-players.

Reply:
I don't remember if I checked this part already, but it has been changed after @aleos89 review https://github.com/rathena/rathena/pull/7998#discussion_r1380562014

Reply:
Ok then lets leave it here for now and we can revisit this later on.